src/main/kotlin/no/nav/tilleggsstonader/sak/infrastruktur/database/DatabaseConfiguration.kt

```diff
@@ -221,4 +228,40 @@ class DatabaseConfiguration : AbstractJdbcConfiguration() {
             return Fil(bytes)
         }
     }
+
+    @ReadingConverter
+    class PGobjectTilVedtaksdataTilsynBarn : Converter<PGobject, VedtaksdataTilsynBarn> {
+
+        override fun convert(pGobject: PGobject): VedtaksdataTilsynBarn {
+            return objectMapper.readValue(pGobject.value!!)
```

Comment:
Har `pgObject` alltid `value`?

Reply:
Slik jeg har tolket det så er det sånn :) 
Men i [abstract-PR](https://github.com/navikt/tilleggsstonader-sak/pull/56/files#diff-8e1b47e6e04b49204827e624a3f6cc939d1b7c00932373d59846ddc5d1f319b7R120)'en så satte jeg til
`pGobject.value?.let { objectMapper.readValue(it, clazz.java) }`